@xen-orchestra/rest-api/src/vdi-snapshots/vdi-snapshot.controller.mts

```diff
@@ -64,7 +84,7 @@ export class VdiSnapshotController extends XapiXoController<XoVdiSnapshot> {
   async exportVdiSnapshotContent(
     @Request() req: ExRequest,
     @Path() id: string,
-    @Path() format: 'vhd' | 'raw'
+    @Path() format: Exclude<SUPPORTED_VDI_FORMAT, 'qcow2'>
```

Comment:
Is there a limit on snapshot export ? I thik it is handled as any other disk export, so if the VDI export support qcow2, the snapshot should also 
and at least on xo5 the qcow2 export format is supported